docs/UserGuide.md

```diff
@@ -506,23 +514,23 @@ If the problem persists, please report the bug to us.
+| **Edit buyer**        | `editbuyer INDEX [-n NAME] [-ph PHONE] [-e EMAIL] [-a ADDRESS] [-r PRICE RANGE] [-c CHARACTERISTICS] [-pr PRIORITY<HIGH, NORMAL, LOW>]`<br> e.g.,`editbuyer 3 -n John Doe -e johndoe@yahoo.com -r 40000-50000 -pr HIGH`                             |
+| **Edit property**     | `editprop INDEX [-n NAME] [-p PRICE] [-a ADDRESS] [-d DESCRIPTION] [-c CHARACTERISTICS] [-o OWNER NAME] [-ph OWNER PHONE]`<br> e.g., `editprop 3 -n Hill Residence -a Block 225 -ph 82001000`                                                       |
+| **List buyers**       | `listbuyers`                                                                                                                                                                                                                                        |
+| **List properties**   | `listprops`                                                                                                                                                                                                                                         |
+| **Find buyers**       | `findbuyers WORD` <br> e.g. `findbuyers John`                                                                                                                                                                                                       |
+| **Find properties**   | `findprops WORD` <br> e.g. `findprops Heng Mui Keng`                                                                                                                                                                                                |
+| **Filter buyers**     | `filterbuyers [-p PRICE] [-c CHARACTERISTICS] [-pr PRIORITY<HIGH, NORMAL, LOW>] [-fuzzy]` <br> e.g. `filterbuyers -p 500000 -c bright; sunny -pr HIGH -fuzzy`                                                                                       |
+| **Filter properties** | `filterprops [-r PRICE RANGE] [-c CHARACTERISTICS] [-o OWNER NAME] [-fuzzy]` <br> e.g. `filterprops -r 500000-1000000 -c bright; sunny -o GARY -fuzzy`                                                                                              |
+| **Sort buyers**       | `sortbuyers [-n NAME<ASC/DESC>] [-r PRICE RANGE<ASC/DESC>] [-pr PRICE<ASC/DESC>] [-t ENTRY TIME<ASC/DESC>]` <br> e.g. `sortbuyers -pr DESC`                                                                                                         |
+| **Sort properties**   | `sortprops [-n NAME<ASC/DESC>] [-p PRICE<ASC/DESC>] [-t ENTRY TIME<ASC/DESC>]` <br> e.g. `sortprops -p DESC`                                                                                                                                        |
```

Comment:
Oh I just noticed that -t is still called Entry Time, I think Ian mentioned a while ago to change them to time of creation so if you don't mind can help to standardise here!!

Reply:
refactored all entry time instances